src/js/media-chrome-button.js

```diff
@@ -91,7 +92,10 @@ class MediaChromeButton extends window.HTMLElement {
 
     this.nativeEl.appendChild(slotTemplate.content.cloneNode(true));
 
-    shadow.appendChild(buttonHTML);
+    this.shadowRoot.appendChild(buttonHTML);
+
+    const { style } = getOrInsertCSSRule(this.shadowRoot, ':host');
+    style.setProperty('display', `var(--${this.localName}-display, inline-flex)`);
```

Comment:
I thought that relying on `localName` is an issue, but it's only an issue if you're trying to override it (had some issues previously, where the localName was getting set, but the switch to strict mode broke things because it disallows it, but loose mode still allows it and just makes it silently fail).